hw_25/resources/pages/login_page.py

```diff
@@ -0,0 +1,22 @@
+"""Login page"""
+
+
+from selenium.webdriver.common.by import By
+from hw_25.resources.pages.base_page import BasePage
+from hw_25.resources.locators.login_locators import LoginLocators
+
+
+class LoginPage(BasePage):
+
+    def go_to_signup(self):
+        self.driver.find_element(By.CSS_SELECTOR, LoginLocators.signup_button).click()
+
+    def login(self, email, password):
+        self.driver.find_element(By.CSS_SELECTOR, LoginLocators.email_input_login).clear()
+        self.driver.find_element(By.CSS_SELECTOR, LoginLocators.email_input_login).send_keys(email)
```

Comment:
You do not need to search the same element several times
```suggestion
        input = self.driver.find_element(By.CSS_SELECTOR, LoginLocators.email_input_login)
        input.clear()
        input.send_keys(email)
```

Reply:
fixed